src/main/java/core/basesyntax/FieldXY.java

```diff
@@ -1,19 +1,38 @@
 package core.basesyntax;
 
-/**
- * <p>На игровом поле находится робот. Позиция робота на поле описывается двумя
- * целочисленным координатами: X и Y. 
- * Ось X смотрит слева направо, ось Y — снизу вверх.(Помните, как рисовали 
- * графики функций в школе?)
- * В начальный момент робот находится в некоторой позиции на поле.
- * Также известно, куда робот смотрит: вверх, вниз, направо или налево.
- * Ваша задача — привести робота в заданную точку игрового поля.
- * Реализуйте метод moveRobot()</p>
- */
 public class FieldXY {
 
     public void moveRobot(Robot robot, int toX, int toY) {
-        //your code
+        Direction directionX;
+        Direction directionY;
+
+        if (robot.getX() < toX) {
+            directionX = Direction.RIGHT;
+        } else {
+            directionX = Direction.LEFT;
```

Comment:
let's optimize this check 
Direction directionX = (robot.getX() < toX) ? Direction.RIGHT : Direction.LEFT;